packages/manager/modules/pci/src/components/hds/controller.js

```diff
@@ -0,0 +1,29 @@
+import { Environment } from '@ovh-ux/manager-config';
+import { HDS_INFO, SUPPORT_LEVEL_INFO } from './hds.constant';
+
+export default class HdsComponentController {
+  constructor() {
+    this.language = Environment.getUser().language;
+    this.hdsInfoLink = this.getHdsInfoLink();
+    this.supportLevelInfoLink = this.getSupportLevelInfoLink();
+  }
+
+  getHdsInfoLink() {
+    return HDS_INFO[this.language] || HDS_INFO[HDS_INFO.DEFAULT];
```

Comment:
Shouldn't the link be handled by subsidiary like it is done elsewhere ? If I refer to the constants language (https://github.com/ovh/manager/pull/4336/files#diff-e288eb5d2429b90b0d19e00100a27ce37460ea54978ecc2569e11fb829ab859dR20) `fr_MA` is not a valid value from our point of view, therefore the customer will never have access to this link

Reply:
As my understanding when a customer create an account he complete language and country fields.
So ```.language``` is more accuracy i think ?

But as i saw on others place we use subsidiary.
I updated my list according the subsidiary.
Thanks.